TestsFoundation/TestsFoundation/Extensions/XCUIElementExtensions.swift

```diff
@@ -19,50 +19,241 @@
 import Foundation
 import XCTest
 
+public var app: XCUIApplication { XCUIApplication() }
+
 public extension XCUIElement {
-    func find(label: String, type: XCUIElement.ElementType = .any) -> Element {
-        return descendants(matching: type).matching(label: label).firstElement
+    static let defaultTimeout: TimeInterval = 15
+    static let defaultGracePeriod: UInt32 = 1
+    var isVisible: Bool { exists }
+    var isVanished: Bool { !(exists && isHittable) }
+    func tacticalSleep(ms: UInt32 = 500000) { usleep(ms) }
```

Comment:
`ms` for me stands for milliseconds but `usleep` actually takes microseconds. I would be more comfortable passing millisecs (ms) and before passing to usleep multiplying by 1000. In a later comment I suggested to use TimeInterval instead of integers so we could consider using it here as well.

Reply:
Done.